pandas/tests/frame/methods/test_explode.py

```diff
@@ -223,9 +223,15 @@ def test_explode_sets():
 
 
 @pytest.mark.parametrize(
-    "input_subset, expected_dict, expected_index",
+    "input_dict, input_index, input_subset, expected_dict, expected_index",
     [
         (
+            {
```

Comment:
I think you should include the github issue on both of your test changes to make sure its clear why you added these tests.